cloudservice/gcp/gcp_test.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package gcp
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestGCP(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "CloudService GCP Suite")
+}
+
+var _ = BeforeSuite(func() {
+})
+
+var _ = AfterSuite(func() {
+})
+
+var _ = Describe("GCP Parsing", func() {
+	Context("Parse GCP IP Range file", func() {
+		ipRangeFile := "{\"syncToken\":\"1000000000\",\"createDate\":\"2022-09-01-22-03-06\",\"prefixes\":[{\"ipv4Prefix\": \"34.80.0.0/15\",\"service\": \"Google Cloud\",\"scope\": \"asia-east1\"},{\"ipv4Prefix\": \"34.80.0.0/15\",\"service\": \"Google Cloud\",\"scope\": \"asia-east1\"},{\"ipv4Prefix\": \"34.80.0.0/15\",\"service\": \"Google Cloud\",\"scope\": \"asia-east1\"},{\"ipv4Prefix\": \"34.80.0.0/15\",\"service\": \"Google Cloud\",\"scope\": \"asia-east1\"}]}"
```

Comment:
This value does not seems representative of what is in:
* https://www.gstatic.com/ipranges/goog.json

Is this intentional?


It seems more representative of this file:
* https://www.gstatic.com/ipranges/cloud.json

That we do not plan to use, am I correct?

Reply:
You're right, thanks for catching that!

I began the implementation with the cloud.json file as it seemed logical, and forgot to replace the data.